src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -3,11 +3,11 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
 using System.Text;
+using static System.Net.Mime.MediaTypeNames;
```

Comment:
Is this necessary?

Reply:
It is not, good catch. I need to update my VS settings to stop doing this automatically.